internal/rbd/rbd_util.go

```diff
@@ -2011,3 +2015,23 @@ func getCephClientLogFileName(id, logDir, prefix string) string {
 
 	return fmt.Sprintf("%s/%s-%s.log", logDir, prefix, id)
 }
+
+// strategicActionOnLogFile act on log file based on cephLogStrategy.
+func strategicActionOnLogFile(ctx context.Context, logStrategy, logFile string) {
+	var err error
+
+	switch strings.ToLower(logStrategy) {
+	case "compress":
+		if err = log.GzipLogFile(logFile); err != nil {
+			log.ErrorLog(ctx, "failed to compress logfile %q: %v", logFile, err)
+		}
```

Comment:
can this be moved to a new goroutine. if it takes more time depends on the data of the log file the performance will be affected.

Reply:
in other words, IMO strategicActionOnLogFile can be moved to go-routine.

Reply:
This is Awful! Actually, my respin was to address your previous comment on similar lines, and I made the changes locally and tested the unit pieces. It is strange it is not reflecting here. Sorry for jumping the gun and marking your previous comment as resolved :-(